src/measure_theory/set_integral.lean

```diff
@@ -464,6 +464,135 @@ lemma continuous_at.integral_sub_linear_is_o_ae
   is_o (λ s, ∫ x in s, f x ∂μ - (μ s).to_real • f a) (λ s, (μ s).to_real) ((𝓝 a).lift' powerset) :=
 (ha.mono_left inf_le_left).integral_sub_linear_is_o_ae hfm (μ.finite_at_nhds a)
 
+section continuous_linear_map
+
+variables {μ : measure α} [normed_group E] [normed_space ℝ E] [normed_group F]  [normed_space ℝ F]
+
+namespace measure_theory
+
+lemma integrable.clm_apply {φ : α → E} (φ_int : integrable φ μ) (L : E →L[ℝ] F) :
+  integrable (λ (a : α), L (φ a)) μ :=
+((integrable.norm φ_int).const_mul ∥L∥).mono' (eventually_of_forall $ λ a, L.le_op_norm (φ a))
+
+variables [second_countable_topology E] [measurable_space E] [borel_space E]
+
+lemma l1.norm_clm_apply_le (φ : α →₁[μ] E) (L : E →L[ℝ] F) : ∀ᵐ a ∂μ, ∥L (φ a)∥ ≤ ∥L∥*∥φ a∥ :=
+eventually_of_forall (λ a, L.le_op_norm (φ a))
+
+section
+variables [measurable_space F] [borel_space F] [second_countable_topology F]
+
+/-- Composing `φ : α →₁[μ] E` with `L : E →L[ℝ] F`. -/
+def l1.clm_apply (φ : α →₁[μ] E) (L : E →L[ℝ] F) : α →₁[μ] F :=
+l1.of_fun (λ a, L (φ a)) (φ.measurable.clm_apply L) (φ.integrable.clm_apply L)
+
+lemma l1.clm_apply_apply (φ : α →₁[μ] E) (L : E →L[ℝ] F) : ∀ᵐ a ∂μ, (φ.clm_apply L) a = L (φ a) :=
+l1.to_fun_of_fun _ _ _
+
+lemma l1.measurable_clm_apply (L : E →L[ℝ] F) (φ : α →₁[μ] E) : measurable (φ.clm_apply L) :=
+(φ.clm_apply L).measurable
+
+lemma l1.integrable_clm_apply (L : E →L[ℝ] F) (φ : α →₁[μ] E) : integrable (φ.clm_apply L) μ :=
+(φ.clm_apply L).integrable
+
+end
+
+lemma l1.integrable_clm_apply' (L : E →L[ℝ] F) (φ : α →₁[μ] E) : integrable (λ a, L (φ a)) μ :=
+φ.integrable.clm_apply L
+
+variables [measurable_space F]
+
+lemma l1.measurable_clm_apply' [borel_space F] (L : E →L[ℝ] F) (φ : α →₁[μ] E) :
+  measurable (λ a, L (φ a)) := L.measurable.comp φ.measurable
+
+variables [borel_space F] [second_countable_topology F]
+
+lemma l1.integral_clm_apply [complete_space F] (φ : α →₁[μ] E) (L : E →L[ℝ] F):
+  ∫ a, (φ.clm_apply L) a ∂μ = ∫ a, L (φ a) ∂μ :=
+by simp [l1.clm_apply]
+
+/-- Composing `φ : α →₁[μ] E` with `L : E →L[ℝ] F`, seen as a `ℝ`-linear map on `α →₁[μ] E`. -/
+def l1.clm_applyₗ (L : E →L[ℝ] F) : (α →₁[μ] E) →ₗ[ℝ] (α →₁[μ] F) :=
```

Comment:
It feels strange to have this one in the namespace `measure_theory.l1`. I'd rather have it in `continuous_linear_map`, just like `continuous_linear_map.l1_apply` (maybe with more similar names)

Reply:
I changed all names and namespaces. I'm sorry I opened this PR without doing the name cleanup. I wanted Yury to have it and I was too fast.